src/Network/HTTP2/Client.hs

```diff
@@ -282,7 +282,7 @@ type PushPromiseHandler =
 -- @ _ <- (withHttp2Stream myClient $ \stream -> StreamDefinition _ _) @
 --
 -- Please refer to 'StreamStarter' and 'StreamDefinition' for more.
-withHttp2Stream :: Http2Client -> StreamStarter a
+withHttp2Stream :: Http2Client -> forall a. StreamStarter a
```

Comment:
```suggestion
withHttp2Stream :: forall a . Http2Client -> StreamStarter a
```
it is more common to qualify type variables at the outside. Is there a reason not to do that here?

Reply:
This did not work, unfortunately:

<img width="758" alt="image" src="https://user-images.githubusercontent.com/51825/167415645-4ac9b7d0-be7b-485b-9c1c-a5e643e6f3af.png">